win32ss/printing/base/spoolsv/init.c

```diff
@@ -8,7 +8,8 @@
 #include "precomp.h"
 
 DWORD
-_RpcSpoolerInit()
+WINAPI
```

Comment:
Technically this should not matter, but lgtm (the VOID does matter)

Reply:
The VOID doesn't matter either FWIW ;)  It's only relevant on declarations, not definitions.